test/versioned/mongodb/db.tap.js

```diff
@@ -63,20 +63,18 @@ if (semver.satisfies(mongoPackage.version, '<3')) {
 }
 
 dbTest('addUser, authenticate, removeUser', [], function addUserTest(t, db, verify) {
-  var user = null
-  var userName = 'user-test'
-  var userPass = 'user-test-pass'
+  const userName = 'user-test'
+  const userPass = 'user-test-pass'
 
   db.removeUser(userName, function preRemove() {
     // Don't care if this first remove fails, it's just to ensure a clean slate.
     db.addUser(userName, userPass, {roles: ['readWrite']}, added)
   })
 
-  function added(err, _user) {
```

Comment:
Did the call back signature change?

Reply:
yea so it no longer returns data, just `{ ok: 1 }`. I figured the assertions later down the chain were pointless anyways